### ISSUE TITLE
Console error: Focus handling does not check for config form properly

### ISSUE THREAD

**Issue by ChristophHi** [SAP/spartacus#12341]
Configure coffee machine. Select values, tab through groups, and add configuration to cart. Configure cart entry configuration.
From time to time (not exactly reproducible) a console error appears when navigating to OV page:

core.js:4442 ERROR TypeError: Cannot read property 'querySelectorAll' of null
    at SelectFocusUtility.query (select-focus.util.ts:33)
    at SelectFocusUtility.findFocusable (select-focus.util.ts:70)
    at KeyboardFocusService.findFocusable (tab-focus.service.ts:141)
    at ConfiguratorStorefrontUtilsService.focusFirstAttribute (configurator-storefront-utils.service.ts:122)
    at SafeSubscriber._next (configurator-previous-next-buttons.component.ts:96)
    at SafeSubscriber.__tryOrUnsub (Subscriber.js:183)
    at SafeSubscriber.next (Subscriber.js:122)
    at Subscriber._next (Subscriber.js:72)
    at Subscriber.next (Subscriber.js:49)
    at SwitchMapSubscriber.notifyNext (switchMap.js:66)
    at SimpleInnerSubscriber._next (innerSubscribe.js:10)
    at SimpleInnerSubscriber.next (Subscriber.js:49)
    at SwitchMapSubscriber.notifyNext (switchMap.js:66)
    at SimpleInnerSubscriber._next (innerSubscribe.js:10)
    at SimpleInnerSubscriber.next (Subscriber.js:49)

It seems to happen because the configuration form is no longer available. We either need to handle the non-availability of the form, or prevent focusFirstAttributes from being called in that situation
